### DISCUSSION TITLE
Enabeling forecast of next 8 hours tariff

### DISCUSSION THREAD

**Discussion by bosmanrja** [fsaris/home-assistant-zonneplan-one#127] [Q&A] [answered]
How can I enable 8 hours forecast information. It is disabeled and I have registered via my e-mail to zonneplan.

**Comment by fsaris** [comment, marked as answer]
Enable the entity in HA. Edit the entity and enable it

**Reply by tweakable33** [reply]
I only see 'sensor.zonneplan_next_gas_tariff' and no entity for electricy forecast. Any idea?

**Reply by fsaris** [reply]
Check for the `Forecast tariff hour *` sensors or the attributes of the `Current electricity tariff` sensor

**Comment by bosmanrja** [comment]
Hartelijk dank Frans! Zo simpel dat ik mij eigenlijk schaam :-)

Jan-Roel 

⁣Get BlueMail for Android ​
